.travis.yml

```diff
@@ -1,29 +1,45 @@
+sudo: required
```

Comment:
I don't want that apt line, that's the issue :D I theory I could use something like `$(lsb_release -s -c)` but I'm not sure if config file allows something like:

```yaml
sources:
  - sourceline: 'deb http://repo.percona.com/apt $(lsb_release -s -c) testing'
```